### PR TITLE
Built structural framework for Operator-SDK based KubeBot

### LINKED ISSUE #3: kubebot architecture

### LINKED ISSUE #4: Intergrate Dialogflow with kubebot

### PR DESCRIPTION
**Pull request by priyanshuraj400**
Created a custom CRD API for scaffolding the resource API and controller. fix #4 closes #3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,17 @@
...
+
+KubeBot is a smart tool that pulls out of box metrics, traces, events and logs collection for applications running inside Kubernetes and reports all the collected data on a single dashboard and push notifications to users for critical ones. The collected metrics, traces, events and logs will help to debug the application running inside Kubernetes faster and effectively. A main feature of KubeBot is that after collecting traces, it does a root-cause analysis and sends alerts to the user regarding fixing the issue occuring in the cluster.
+
+## Project High Level Design:
```
Comment (swapnalshahil) [inline README.md]: Hi thanks for the PR, can you also add the images in the Readme?

Comment (PratikDhanave) [reply README.md]: @swapnalshahil that a very good suggestion @priyanshuraj400 if you can add architecture diagram in readme file that will be helpful to understand kubebot architecture.

Comment (priyanshuraj400) [reply README.md]: Sure @swapnalshahil @PratikDhanave, I will add the image in next PR

**Comment by priyanshuraj400** [comment]
fixes #4

**Comment by priyanshuraj400** [comment]
fix #13

**Pull request merged**